webpack.config.babel.js

```diff
@@ -122,6 +127,7 @@ const PROD_CONFIG = {
   SMS_DELIVERY_URL: 'https://telephony.onfido.com',
   PUBLIC_PATH: `https://assets.onfido.com/web-sdk-releases/${packageJson.version}/`,
   USER_CONSENT_URL: 'https://assets.onfido.com/consent/user_consent.html',
+  AUTH_URL: 'https://edge.api.onfido.com/v3',
```

Comment:
Is this only available on Prod environment? How about Staging/Dev environment (see `STAGING_CONFIG` [below](https://github.com/onfido/onfido-sdk-ui/blob/f9aaf6e980eddd7130e7e96ffe32af4007747d31/webpack.config.babel.js#L152)) and the `TEST_E2E_CONFIG` for automated tests?

Reply:
e2e tests: https://gitlab.eu-west-1.mgmt.onfido.xyz/onfido/biometrics/auto/facetec_tests/